client/scripts/views/pages/notifications.ts

```diff
@@ -540,6 +540,7 @@ const IndividualCommunityNotifications: m.Component<{
       (s) => (s.OffchainCommunity?.id === community.id || s.Chain?.id === community.id)
         && s.category !== NotificationCategories.NewThread
         && s.category !== NotificationCategories.NewMention
+        && s.category !== NotificationCategories.NewCollaboration
```

Comment:
Would it make more sense to call the notification category `NotificationCategories.NewCollaborators`? even though the number of new collaborators could be one or many